opensrp-chw/src/main/java/org/smartregister/chw/activity/ChildHomeVisitActivity.java

```diff
@@ -55,7 +59,31 @@ public void submitVisit() {
                 } catch (Exception e) {
                     Timber.e(e);
                 }
+            }
 
+            String childMinorAilmentStringEn = "Child Minor Ailments";
```

Comment:
@issyzac - could we use getStringResources here from Android to handle the translation?